libs/eo/emissions/shell/src/lib/eo-emissions-page-lead-by-example.component.ts

```diff
@@ -0,0 +1,75 @@
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { ChangeDetectionStrategy, Component, NgModule } from '@angular/core';
+import { MatCardModule } from '@angular/material/card';
+
+@Component({
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  selector: 'eo-emissions-page-lead-by-example',
+  styles: [
+    `
+      :host {
+        display: block;
+      }
+
+      mat-card {
+        background: var(--watt-color-neutral-white);
+        img {
+          display: block;
+          width: 544px; // Magic number by designer
+          height: 201px; // Magic number by designer
+        }
```

Comment:
```suggestion
        display: grid;
        gap: var(--watt-space-m);
        background: var(--watt-color-neutral-white);

        img {
          width: 544px; // Magic number by designer
          height: 201px; // Magic number by designer
        }
```

Reply:
This will create the gap between things without needing the stack class :)

Reply:
This will not work, as it would add the grid-gap between all the elements in the content area - Instead we should rely on the line height of the base elements styling (from watt)